thirdeye-ui/src/app/rest/anomalies/anomalies.rest.ts

```diff
@@ -39,6 +39,7 @@ export const getAnomalies = async ({
     dataset,
     metric,
     enumerationItemId,
+    showAllAnomalies = false,
```

Comment:
rename to `filterIgnoredAnomalies` and default to true to be more concise

Reply:
Updated accordingly